ocaml/xapi/xapi_vlan.ml

```diff
@@ -13,6 +13,7 @@
  *)
 module D = Debug.Make(struct let name="xapi" end) 
 open D
+open Network
```

Comment:
I'd prefer a local `let open Network in` where you need it to keep the code readable without tool support.